File: hyunji/Programmers/lv0_최댓값_만들기_(2).py
def solution(numbers):
    n = list(map(int, numbers))
    n = sorted(n)
    for i in range(len(numbers)):
        mul = n[i] * n[i-1]
        mul2 = n[-i] * n[-i-1]
    if mul > mul2:
        answer = mul
    else:
        answer = mul2
    return answer

File: seongwon/Programmers/Level_0/Lv0_최댓값_만들기_(2).cpp
#include <string>
#include <vector>
#include <iostream> 
#include <algorithm>

using namespace std;

int solution(vector<int> numbers) {
    int answer = 0;
    int l = numbers.size();
    sort(numbers.begin(), numbers.end());
    if(numbers[0]*numbers[1] > numbers[l-2]*numbers[l-1])
        answer = numbers[0]*numbers[1];
    else
        answer = numbers[l-2]*numbers[l-1];
    return answer;
}